python/ml4ir/applications/classification/pipeline.py

```diff
@@ -168,6 +170,33 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def finish(self, job_status, job_info):
+        # This will write the _SUCCESS/_FAILURE files
+        super().finish(job_status, job_info)
+
+        # Create model_bundle.zip
+        if self.args.file_handler == FileHandlerKey.SPARK:
+            # Copy tfrecord files in model_files
+            os.makedirs("epmm")
+            shutil.copytree(os.path.join(self.models_dir_local, "final/tfrecord/"), "model_files")
+            shutil.make_archive("./epmm/model_bundle", "zip", "model_files")
+
+            (self.file_io
+             .read_df(self.feature_config.get_label()['feature_layer_info']['args']['vocabulary_file'])
+             .to_csv("key_prefix_labels.csv"))
+            shutil.make_archive("./epmm/key_prefix_labels.csv", "zip", ".", "key_prefix_labels.csv")
+
+        # Create org_id_to_key_prefix.csv.zip
+            org_id = [x for x in self.feature_config.get_all_features() if x['name'] == 'organization_id'][0]
+            org_mapping = org_id['feature_layer_info']['args']['vocabulary_file']
```

Comment:
what does "org_id" mean to an open source project like this?  Remember, this is *generic* "ml4ir" - the code in here cannot specify any SFDC-internal terminology, and file names need to come in via cmdline args or in the config files.

Reply:
yes, I will do.